lib/ocpp/v201/smart_charging.cpp

```diff
@@ -21,6 +21,19 @@ ProfileValidationResultEnum SmartChargingHandler::validate_evse_exists(int32_t e
                                               : ProfileValidationResultEnum::Valid;
 }
 
+ProfileValidationResultEnum
+SmartChargingHandler::validate_charge_point_max_profile(const ChargingProfile& profile) const {
```

Comment:
@folkengine I am confused. I see the new method, but where is it invoked from?

Reply:
Once the individual validations are written they will be added to a switchboard style general function that includes validations that apply to all profiles. For this card, it doesn't matter. That's TBD.